Sofa/Sofa/EventView.swift

```diff
@@ -0,0 +1,210 @@
+        divider.image = UIImage(named: "Divider Horizontal")
+        
+        if let homeLogoUrl = event.homeTeam.logoUrl, let url = URL(string: homeLogoUrl) {
+            homeTeamLogo.loadImage(from: url)
+        }
+        if let awayLogoUrl = event.awayTeam.logoUrl, let url = URL(string: awayLogoUrl) {
+            awayTeamLogo.loadImage(from: url)
+        }
+        
+        homeTeamLabel.text = event.homeTeam.name
+        awayTeamLabel.text = event.awayTeam.name
+        
+        
+        if let homeScore = event.homeScore {
+            homeScoreLabel.text = "\(homeScore)"
+        }
+
+        if let awayScore = event.awayScore {
+            awayScoreLabel.text = "\(awayScore)"
+        }
+        
+        let (timeText, timeColor,Live) = calculateMatchTime(startTimestamp: event.startTimestamp)
+        timeLabel.text = timeText
+        timeLabel.textColor = timeColor
+        
+        if Live {
+            homeScoreLabel.textColor = .blue
+            awayScoreLabel.textColor = .blue
+        } else {
+            homeScoreLabel.textColor = .black
+            awayScoreLabel.textColor = .black
+        }
+        
+        let formatter = DateFormatter()
+        formatter.dateFormat = "HH:mm"
+
+        let startDate = Date(timeIntervalSince1970: TimeInterval(event.startTimestamp))
+        start.text = formatter.string(from: startDate)
+        
+
+    }
+    
+    override func setupConstraints() {
+        homeTeamLogo.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(80)
+            make.top.equalToSuperview().offset(10)
+            make.width.height.equalTo(16)
+        }
+        homeTeamLabel.snp.makeConstraints { make in
+            make.height.equalTo(16)
+            make.width.equalTo(192)
+            make.leading.equalToSuperview().offset(104)
+            make.top.equalToSuperview().offset(10)
+        }
+        
+        homeScoreLabel.snp.makeConstraints { make in
+            make.height.equalTo(16)
+            make.width.equalTo(32)
+            make.trailing.equalToSuperview().offset(16)
+            make.top.equalToSuperview().offset(10)
+        }
+        
+        awayTeamLabel.snp.makeConstraints { make in
+            make.height.equalTo(16)
+            make.width.equalTo(192)
+            make.leading.equalToSuperview().offset(104)
+            make.top.equalToSuperview().offset(30)
+        }
+        awayTeamLogo.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(80)
+            make.top.equalToSuperview().offset(30)
+            make.width.height.equalTo(16)
+        }
+        awayScoreLabel.snp.makeConstraints { make in
+            make.height.equalTo(16)
+            make.width.equalTo(32)
+            make.trailing.equalToSuperview().offset(16)
+            make.top.equalToSuperview().offset(30)
+        }
+        divider.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(63)
```

Comment:
Nakacit se na prvi lijevi view. Onaj koji ti "vise" fixan, to je startLabel. i offsetaj se koliko treba.

Inace 63 nije dobra vrijednost (mozda se dizajner zeznio) idealno je da je vrijednost offseta djeljiva sa 8, ovo bi onda bilo 64 vrijednost koju zelimo.